### PR TITLE
NMS-9401: update release notes documentation

### PR DESCRIPTION
**Pull request by RangerRick**
This PR resolves the NMS-9401 issue about future breaking XML changes in Horizon 21.  It also includes a merge of the doc updates from 19.1, and the release notes for Horizon 20 since I was already working in that document anyways.  :)

* JIRA: http://issues.opennms.org/browse/NMS-9401

Our [continuous integration system](http://bamboo.internal.opennms.com:8085) will test and verify your changes.

### CODE REVIEW

File: opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew-19.adoc [hunk 1]
```diff
@@ -38,10 +43,13 @@ If you use the Remote Poller, you will need to install the `opennms-webapp-remot
 * *Automatic provisioning of Minions*: Minion devices will be added to the _Minions_ requisition and monitored for connectivity with OpenNMS.
 * *GraphML Topology Provider*: A new Topology Provider is available that reads GraphML formatted graphs and displays them in the the Topology UI.
 * *OpenNMS Elasticsearch ReST Forwarder*: This Elasticsearch forwarder can store events, alarms, and alarm state changes in Elasticsearch for archival or searching. It is compatible with all currently supported versions of Elasticsearch.
+* *Drools Persistence*: Drools working memory will now be persisted to disk upon shutdown of OpenNMS. (new in 19.1)
```
Comment (j-white) [inline opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew-19.adoc]: will vs can

File: opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew-20.adoc [hunk 2]
```diff
@@ -13,21 +13,73 @@
...
   For a detailed list of all removed events see link:https://issues.opennms.org/browse/NMS-9303[NMS-9303].
-  For people maintaining databases prior OpenNMS 1.8, we provide a maintenance script for Linux and Windows in the _opennms-contrib_ package to clean up old events in the database.
-  The script is named `NMS-9303-cleanup-deprecated-events.sh}` and executes a SQL script.
+  For people maintaining databases prior to OpenNMS 1.8, we provide a maintenance script for Linux and Windows in the _opennms-contrib_ package to clean up old events in the database.
+  The script is named `NMS-9303-cleanup-deprecated-events.sh` and executes a SQL script.
   In case you run PostgreSQL on Windows, you can use `psql.exe -d opennms -U opennms -W -a -f NMS-9303-cleanup-deprecated-events.sql` to run the maintenance script.
 
+=== Future Breaking Changes
+
+* *Configuration File Format*: In OpenNMS Horizon 21, we will (finally) be re-formatting all XML
```
Comment (j-white) [inline opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew-20.adoc]: Speaking of format, shouldn't we stick to the one sentence per line in these files too? :)

Comment (RangerRick) [inline opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew-20.adoc]: "too"?  I take it this was mentioned elsewhere I wasn't looking?  :)   But yeah, I can...

File: opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew-19.adoc (hunk 1 above)
Comment (RangerRick) [inline opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew-19.adoc]: Are you saying it should be "can"?  Does it not do it by default?

Comment (j-white) [inline opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew-19.adoc]: Yeah, it's optional, and off by default.

**Pull request merged**